app/src/androidTest/java/com/sdp/movemeet/view/chat/ChatActivitySendImageTest.java

```diff
@@ -0,0 +1,71 @@
+package com.sdp.movemeet.view.chat;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+
+import androidx.test.espresso.intent.rule.IntentsTestRule;
+
+import com.google.firebase.storage.FirebaseStorage;
+import com.google.firebase.storage.StorageReference;
+import com.sdp.movemeet.R;
+import com.sdp.movemeet.view.profile.EditProfileActivity;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import static junit.framework.Assert.assertTrue;
+
+public class ChatActivitySendImageTest {
```

Comment:
Nice test for the image in chat !

Reply:
Yes, I implemented it this morning and it seems to work... Normally, with this, we can increase the coverage for this Android activity!